docker-standalone-ethereum-studio-on-ubuntu/README.md

```diff
@@ -0,0 +1,3 @@
+# Deployment of Ethereum Studio docker in standalone mode
```

Comment:
Let's name this folder ethereum-studio-docker-standalone-ubuntu since ethereum studio seems to be the main scenario here